resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/SecretsTestsSuite.scala

```diff
@@ -72,7 +72,7 @@ private[spark] trait SecretsTestsSuite { k8sSuite: KubernetesSuite =>
           doBasicExecutorPodCheck(executorPod)
           checkSecrets(executorPod)
         },
-        appArgs = Array("1000") // give it enough time for all execs to be visible
+        appArgs = Array("10000") // give it enough time for all execs to be visible
```

Comment:
why change these?

Reply:
On a slow cluster these timeouts are too short.  When testing this against our custom named images on a low spec dev cluster this test repeatedly failed because the executors simply didn't launch fast enough

Reply:
Yes that is correct on our ci we have increased that too, but on minikube is fine.

Reply:
Removed this change from this PR, will submit separately